src/App.js

```diff
@@ -1,13 +1,92 @@
-import React from 'react';
-import './App.css';
-
-function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
-}
-
-export default App;
-
-
-
+import React from 'react';
+import './App.css';
+import bag1 from "./assets/bag_1.png";
+import bag2 from "./assets/bag_2.png";
+import bag3 from "./assets/bag_3.png";
+import bag4 from "./assets/bag_4.png";
+import brand from "./assets/brand.png"
+import our_story from "./assets/our_story.png"
+import Nav from './Nav';
+import Product from './Product'
+import Tiles from './Tiles'
+function App() {
+  return ( <>
+        <header>
+          <h1>Handbags & Purses</h1>
+        </header>
+              <nav>
+            <Nav
+                title={"to the collection"}
+            />
+            <Nav
+                title={"shop all bags"}
+            />
+            <Nav
+                disabled={"disabled"}
+                title={"pre-orders"}
+            />

```

Comment:
het verschil tussen nav en Nav is minimaal en daardoor moeilijk te zien ook wordt er niet ingesprongen (tip Ctrl Alt L op een windows/linux pc doet wonderen) Wat dit stuk voor mij erg moeilijk leesbaar maakt.
Zorg voor goed onderscheidende tags bv NavButton of Button zijn goed te onderscheiden tov nav. Nav op zich is geen verkeerde naam alleen zo kort op nav maakt het onduidelijk.
En zorg voor inspringen zodat je goed kan zien wat waar gesloten wordt en wat bij elkaar hoort (Webstorm Reformat Code doet wonderen hierin)